### PR TITLE
Take topk from config

### PR DESCRIPTION
**Pull request by dumbPy**
hardcoding topk to 3,5 isn't generalized enough since it doesn't allow training on datasets below num_classes = 5 
This helped me train a model with num_classes below 5.

### CODE REVIEW

**Comment by innerlee** [comment]
@kennymckormick why they are computed in the loss function, are they part of the loss computation?

**Comment by irvingzhang0512** [comment]
BTW, this issue is fixed in #555

**Comment by kennymckormick** [comment]
> @kennymckormick why they are computed in the loss function, are they part of the loss computation?

Cuz we want to see current top-k accuracy during AVA training. Here `top-k` indicates: the accuracy for top k action labels predicted (since AVA is multi-label).

File: mmaction/models/heads/bbox_head.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ class BBoxHeadAVA(nn.Module):
             'max'. Default: 'max'.
         in_channels (int): The number of input channels. Default: 2048.
         num_classes (int): The number of classes. Default: 81.
+        topk (Union[int, Tuple[int]]): k values to calculate topk accuracies. Default: (3, 5).
```
Comment (kennymckormick) [inline mmaction/models/heads/bbox_head.py:25]: I think the type of topk should be tuple[int], since we have `for k in topk`

Comment (dumbPy) [reply mmaction/models/heads/bbox_head.py:25]: It get's passed to `multilabel_accuracy` where it's handled.
```python
    def multilabel_accuracy(self, pred, target, topk=1, thr=0.5):
        if topk is None:
            topk = ()
        elif isinstance(topk, int):
            topk = (topk, )
```
So for the result for loop, I should add the same `isinstance` handling. But since this is already in #555  I am going to close the PR anyways.

**Comment by OpenMMLab-Coodinator** [comment]
Hi @dumbPy ！We are grateful for your efforts in helping improve this open-source project during your personal time. 
Welcome to join OpenMMLab Special Interest Group (SIG) , where you can share your experiences, ideas, and build connections with like-minded peers. 

To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤
